src/style.css

```diff
@@ -1,3 +1,154 @@
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
 }
+
+.card {
+  padding: 32px 19px 19px;
+  display: flex;
+  flex-direction: column;
+  width: 198px;
+  height: 406px;
+  align-items: center;
+  flex-shrink: 0;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  background: #fff;
+}
+
+.card__img {
+  width: 160px;
+  height: 134px;
+  /* margin-bottom: 40px; */
+  flex-shrink: 0;
+}
+
+.card__content {
+  margin-top: 28px ;
+}
+
+.card__name {
+  margin-bottom: 4px;
```

Comment:
try not to mix bottom and top margins, use only one

Reply:
not fixed
https://github.com/mate-academy/layout_product-cards/pull/3323#discussion_r1267630271